bool call
---
float32 error
float32 dist_door
